### ISSUE TITLE
GuardDuty examples fails due to init command not adding AWSAccount.RootEmail for MasterAccount 

### ISSUE THREAD

**Issue by craighurley** [org-formation/org-formation-cli#86]
When deploying guardduty based on the example guardduty and task and template:

- tasks: https://github.com/OlafConijn/AwsOrganizationFormation/blob/5a1606bf30c1092510bf88eee379345f9ad2d46e/examples/organization-tasks.yml#L14-L20
- guardduty: https://github.com/OlafConijn/AwsOrganizationFormation/blob/5a1606bf30c1092510bf88eee379345f9ad2d46e/examples/templates/guardduty.yml

... it fails with the following error:

```
$ org-formation perform-tasks ./tasks.yaml --profile $AWS_PROFILE --print-stack
...
INFO: Executing: update-stacks templates/guardduty.yaml guardduty.
ERROR: error updating CloudFormation stack guardduty in account 111111111111 (us-west-2). 
Resource is not in the state stackCreateComplete (111111111111 = MasterAccount)
ERROR: Resource Member222222222222 failed because Resource creation cancelled. (222222222222 = ProdAccount)
ERROR: Resource Member333333333333 failed because Resource creation cancelled. (333333333333 = IdentityAccount)
ERROR: Resource Member444444444444 failed because Resource creation cancelled. (444444444444 = SharedservicesAccount)
ERROR: Resource Member555555555555 failed because Resource creation cancelled. (555555555555 = BackupAccount)
ERROR: Resource FindingsTopic failed because Invalid parameter: Endpoint (Service: AmazonSNS; Status Code: 400; Error Code: InvalidParameter; Request ID: b2775713-81c0-541f-af60-00a791aee451; Proxy: null).
ERROR: Stack guardduty in account 111111111111 (us-west-2) update failed. reason: Resource is not in the state stackCreateComplete (111111111111 = MasterAccount)
Resource is not in the state stackCreateComplete
ResourceNotReady: Resource is not in the state stackCreateComplete
    at constructor.setError (/usr/local/lib/node_modules/aws-organization-formation/node_modules/aws-sdk/lib/resource_waiter.js:182:47)
    at Request.CHECK_ACCEPTORS (/usr/local/lib/node_modules/aws-organization-formation/node_modules/aws-sdk/lib/resource_waiter.js:44:12)
    at Request.callListeners (/usr/local/lib/node_modules/aws-organization-formation/node_modules/aws-sdk/lib/sequential_executor.js:106:20)
    at Request.emit (/usr/local/lib/node_modules/aws-organization-formation/node_modules/aws-sdk/lib/sequential_executor.js:78:10)
    at Request.emit (/usr/local/lib/node_modules/aws-organization-formation/node_modules/aws-sdk/lib/request.js:683:14)
    at Request.transition (/usr/local/lib/node_modules/aws-organization-formation/node_modules/aws-sdk/lib/request.js:22:10)
    at AcceptorStateMachine.runTo (/usr/local/lib/node_modules/aws-organization-formation/node_modules/aws-sdk/lib/state_machine.js:14:12)
    at /usr/local/lib/node_modules/aws-organization-formation/node_modules/aws-sdk/lib/state_machine.js:26:10
    at Request.<anonymous> (/usr/local/lib/node_modules/aws-organization-formation/node_modules/aws-sdk/lib/request.js:38:9)
    at Request.<anonymous> (/usr/local/lib/node_modules/aws-organization-formation/node_modules/aws-sdk/lib/request.js:685:12)
WARN: 
WARN: ========================
WARN: Done performing task(s): 7 failed but did not exceed tolerance for failed tasks 10
WARN: Following tasks failed: 
WARN:  - Stack guardduty in account 111111111111 (us-west-2) (111111111111 = MasterAccount)
WARN:  - Stack guardduty in account 333333333333 (us-west-2) (333333333333 = IdentityAccount)
WARN:  - Stack guardduty in account 999999999999 (us-west-2) (999999999999 = SecurityAccount)
WARN:  - Stack guardduty in account 444444444444 (us-west-2) (444444444444 = SharedservicesAccount)
WARN:  - Stack guardduty in account 555555555555 (us-west-2) (555555555555 = BackupAccount)
WARN:  - Stack guardduty in account 666666666666 (us-west-2) (666666666666 = SandboxAccount)
WARN:  - Stack guardduty in account 222222222222 (us-west-2) (222222222222 = ProdAccount)
WARN: ========================
WARN: 
INFO: Task GuardDuty execute successful.
...
```

tasks.yaml:
```
GuardDuty:
  Type: update-stacks
  Template: ./templates/guardduty.yaml
  StackName: guardduty
  StackDescription: Guardduty example template
  MaxConcurrentStacks: 10
  FailedStackTolerance: 10
```

Version:
```
$ org-formation --version
0.9.6
```

To make it work, I changed the email address used by the topic endpoint here:

https://github.com/OlafConijn/AwsOrganizationFormation/blob/5a1606bf30c1092510bf88eee379345f9ad2d46e/examples/templates/guardduty.yml#L42

The difference between failed and passing email address was:

Fail: `!GetAtt AWSAccount.RootEmail` = `string+accountdescription@example.com`
Pass: `string@example.com`

... without looking at the code, my guess is the `+` character is tripping up parsing somewhere?

**Comment by craighurley** [comment]
I just double checked my organization file and it looks like this:

```
AWSTemplateFormatVersion: '2010-09-09-OC'
Description: default template generated for organization with master account 111111111111

Organization:
  MasterAccount:
    Type: OC::ORG::MasterAccount
    Properties:
      AccountName: master
      AccountId: '111111111111'
      Alias: craighurley-master
      PasswordPolicy: !Ref PasswordPolicy
      Tags:
        budget-alarm-threshold: '10'
        account-owner-email: string@example.com

  IdentityAccount:
    Type: OC::ORG::Account
    Properties:
      AccountName: identity
      AccountId: '222222222222'
      RootEmail: string+identity@example.com
      Alias: craighurley-identity
      PasswordPolicy: !Ref PasswordPolicy
      Tags:
        account-owner-email: string@example.com
```

... there is no `RootEmail`, so maybe that caused the error, not my original guess.

However, running `org-formation init organization.yaml --region $ORG_REGION --profile $AWS_PROFILE` does not include the RootEmail key+value.

I run `org-formation init init` as an IAM user with administrator privileges, not the root user.

**Comment by OlafConijn** [comment]
hi Craig,

thanks for this issue. I indeed think it makes sense to add the RootEmail.

Olaf

**Comment by OlafConijn** [comment]
probably could also have been a more descriptive error message :)

**Comment by OlafConijn** [comment]
fixed by changing init command. fixed in v0.9.7
